packages/client/src/routes.ts

```diff
@@ -47,7 +48,7 @@ export function iterateRoutes(routes: IRoutes, paths: Paths[] = []): Paths[] {
 
   if ($routes) {
     for (const route of $routes) {
-      const [href, name, sidebar] = Array.isArray(route) ? route : [route, route];
+      const [path, href, name, sidebar] = Array.isArray(route) ? route : [route, route];
```

Comment:
here we smuggle in the actual `.mdx` file path, so we can later do a lookup for constructing the correct "edit this page on github"-url.